pkg/types/defaults/installconfig.go

```diff
@@ -109,8 +109,10 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 		}
 	case c.Platform.PowerVS != nil:
 		powervsdefaults.SetPlatformDefaults(c.Platform.PowerVS)
-		c.Networking.MachineNetwork = []types.MachineNetworkEntry{
-			{CIDR: *powervsdefaults.DefaultMachineCIDR},
+		if len(c.Networking.MachineNetwork) == 0 {
+			c.Networking.MachineNetwork = []types.MachineNetworkEntry{
+				{CIDR: *powervsdefaults.DefaultMachineCIDR},
+			}
```

Comment:
Let's move this to the block in lines [35-39](https://github.com/openshift/installer/pull/9254/files#diff-6d766497f96d73303cc5919edf8e6a42cb37f917d929722cb30dc3c017824580R35-R39) and leave this switch for the `c.Platform.${platform}` defaulting.